### PR TITLE
implement user authentication using laravel built-in methods

### PR DESCRIPTION
**Pull request by mariashakleva**

### CODE REVIEW

File: server.php [hunk 1]
```diff
@@ -1,21 +0,0 @@
...
-);
-
-// This file allows us to emulate Apache's "mod_rewrite" functionality from the
-// built-in PHP web server. This provides a convenient way to test a Laravel
-// application without having installed a "real" web server software here.
-if ($uri !== '/' && file_exists(__DIR__.'/public'.$uri)) {
-    return false;
-}
-
-require_once __DIR__.'/public/index.php';
```
Comment (davorminchorov) [inline server.php]: Why was this file deleted? I think it is being used when you use the command `php artisan serve`

Comment (mariashakleva) [reply server.php]: I am using Laragon, so I think Avast made some problems, it may treat the file server.php as a threat and automatically deleted it. I added it to the exclusions, so it should fix the problem for future work.

Comment (davorminchorov) [reply server.php]: ok cool, I'll bring it back
